src/prebid.js

```diff
@@ -664,6 +682,7 @@ $$PREBID_GLOBAL$$.getAllWinningBids = function () {
  *   function instead. This function will be removed in Prebid 1.0.
  */
 $$PREBID_GLOBAL$$.buildMasterVideoTagFromAdserverTag = function (adserverTag, options) {
+  utils.logWarn('$$PREBID_GLOBAL$$.buildMasterVideoTagFromAdserverTag will be removed in Prebid 1.0. Include the dfpVideoSupport module in your build, and use the $$PREBID_GLOBAL$$.adserver.buildVideoAdUrl (if DFP is your only Ad Server) or $$PREBID_GLOBAL$$.adservers.dfp.buildVideoAdUrl (if you use other Ad Servers too) function instead.');
```

Comment:
This isn't accurate. `$$PREBID_GLOBAL$$.adservers.dfp.buildVideoAdUrl` is always the function to use.